core/autoscaling.go

```diff
@@ -205,6 +259,7 @@ func (a *autoScalingGroup) cronEventAction() runer {
 		}
 
 		a.loadLaunchConfiguration()
+		a.loadLaunchTemplate()
```

Comment:
why do we return anything on `a.loadLaunchTemplate` if we ignore both its return values?

Reply:
I was using loadLaunchConfiguration as my template, looks like it has the same issue. In both cases only the tests use the return values. I can change it if you like though.